R/tt_showmethods.R

```diff
@@ -156,6 +156,56 @@ setMethod(
   }
 )
 
+
+#' Display columntree structure
+#'
+#' Displays the tree structure of the columns of a
+#' table or column structure object.
+#'
+#' @inheritParams gen_args
+#'
+#' @return Nothing, called for its side effect of displaying
+#' a summary to the terminal.
+#'
+#' @examples
+#' lyt <- basic_table() %>%
+#'   split_cols_by("ARM") %>%
+#'   split_cols_by("STRATA1") %>%
+#'   split_cols_by("SEX", nested = FALSE) %>%
+#'   analyze("AGE")
+#'
+#' tbl <- build_table(lyt, ex_adsl)
+#' coltree_structure(tbl)
+#' @export
+coltree_structure <- function(obj) {
```

Comment:
I think we need an extra page to the vignettes with some use-cases

Reply:
I wrote `coltree_structure` because I needed to introspect coltrees during the development process, I don't expect it to be of much of any use to end users, but I'd say it would go wherever `table_structure` is documented in the vignettes, as its basically doing something analogous to that, except for columns instead of rows